docs/src/main/sphinx/installation/try-it.rst

```diff
@@ -0,0 +1,265 @@
+
+    trino:sf100> SHOW COLUMNS FROM customer;
+
+Which returns:
+
+.. code-block:: text
+
+       Column   |     Type     | Extra | Comment
+    ------------+--------------+-------+---------
+     custkey    | bigint       |       |
+     name       | varchar(25)  |       |
+     address    | varchar(40)  |       |
+     nationkey  | bigint       |       |
+     phone      | varchar(15)  |       |
+     acctbal    | double       |       |
+     mktsegment | varchar(10)  |       |
+     comment    | varchar(117) |       |
+    (8 rows)
+
+Run a SQL script
+~~~~~~~~~~~~~~~~
+
+From the interactive terminal you can run SQL queries. Run:
+
+.. code-block:: shell
+
+    trino> SELECT custkey, name, phone, acctbal FROM tpch.sf100.customer LIMIT 7;
+
+Which returns:
+
+.. code-block:: text
+
+     custkey |        name        |      phone      | acctbal 
+    ---------+--------------------+-----------------+---------
+     3750001 | Customer#003750001 | 17-219-461-2765 | 3711.02 
+     3750002 | Customer#003750002 | 18-659-357-4460 | -966.64 
+     3750003 | Customer#003750003 | 21-489-373-2061 | 9557.01 
+     3750004 | Customer#003750004 | 29-489-412-3729 |  742.49 
+     3750005 | Customer#003750005 | 28-522-477-1174 | 2915.28 
+     3750006 | Customer#003750006 | 25-234-691-1349 | 1011.81 
+     3750007 | Customer#003750007 | 27-555-235-7461 | 8396.42 
+    (7 rows)
+
+Call Trino from terminal
+-------------------------
+
+Using the Trino Interactive CLI isn't required. You can call Trino
+directly from your terminal session. Let's look at a few
+examples.
+
+Pass SQL queries to Trino
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the previous section, you learned how to run a SQL query from the
+Trino Interactive CLI. You can also pass a query directly to Trino. From
+the terminal, run:
+
+.. code-block:: shell
+
+    docker exec -it trino trino --execute 'SELECT custkey, name, phone, acctbal FROM tpch.sf100.customer LIMIT 7'
+
+Which returns:
+
+.. code-block:: text
+
+     custkey |        name        |      phone      | acctbal 
+    ---------+--------------------+-----------------+---------
+     3750001 | Customer#003750001 | 17-219-461-2765 | 3711.02 
+     3750002 | Customer#003750002 | 18-659-357-4460 | -966.64 
+     3750003 | Customer#003750003 | 21-489-373-2061 | 9557.01 
+     3750004 | Customer#003750004 | 29-489-412-3729 |  742.49 
+     3750005 | Customer#003750005 | 28-522-477-1174 | 2915.28 
+     3750006 | Customer#003750006 | 25-234-691-1349 | 1011.81 
+     3750007 | Customer#003750007 | 27-555-235-7461 | 8396.42 
+    (7 rows)
+
+Run SQL scripts with Trino
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Two TPCH scripts are included with the sample files for the `O’Reilly
+book Presto: The Definitive
+Guide <https://www.starburst.io/oreilly-presto-guide-download/>`__.
+
+To use these scripts, download the book’s samples from their `GitHub
+location <https://github.com/trinodb/presto-the-definitive-guide>`__
+either as a zip file or a git clone.
+
```

Comment:
There are several steps left out here. 

- Once the zip is downloaded, it must be unzipped locally.
- If the reader copies the Git URL of the book sample's project, s/he must use a `git clone` command to get a local copy
- cd into the `tpch` directory of the local unzipped or cloned set
- Replace `filename.sql` with `nations.sql`

Only then can the book's sample script work. Otherwise, we might as well just post a short sample.sql on this page for readers to download and run.

Reply:
@Ordinant - For simplicity sake, can we pass in a URI to a SQL script?